bullet/src/BulletCollision/CollisionDispatch/btCollisionDispatcher.cpp

```diff
@@ -189,7 +189,7 @@ bool	btCollisionDispatcher::needsCollision(const btCollisionObject* body0,const
 
 	if ((!body0->isActive()) && (!body1->isActive()))
 		needsCollision = false;
-	else if (!body0->checkCollideWith(body1))
+	else if ((!body0->checkCollideWith(body1)) || (!body1->checkCollideWith(body0)))
```

Comment:
for changes inside bullet code, please add a comment `// XXX EMSCRIPTEN` with an explanation of what changed.

or for a line like this, even
```
#ifdef __EMSCRIPTEN__
..changed line..
#else
..original line..
#endif
```

These will make it much easier to upgrade in the future.

Reply:
that's from https://github.com/bulletphysics/bullet3/commit/7151865c16ba996996206e1fd7869cbb1e7edd8d

Reply:
that's from https://github.com/bulletphysics/bullet3/commit/7151865c16ba996996206e1fd7869cbb1e7edd8d